### PR TITLE
Fixes a bug with lolipop pouches

### PR DESCRIPTION
**Pull request by SpaceLoveSs13**
## About The Pull Request

Fixes bug #7580 

## Why It's Good For The Game

Fixes bug

## Changelog
:cl: SpaceLove
fix: fixed bug with lolipop pouches
/:cl:

### CODE REVIEW

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 1]
```diff
@@ -898,7 +898,7 @@
 			/obj/item/storage/pouch/firstaid = -1,
 			/obj/item/storage/pouch/syringe = -1,
 			/obj/item/storage/pouch/medkit = -1,
-			/obj/item/storage/pouch/med_lolipops = -1,
```
Comment (QualityVan) [inline code/game/objects/machinery/vending/marine_vending.dm:901]: This'll put infinite full pouches in the surplus gear vendors, you should probably change the ones in code/_DEFINES/loadout instead.

Comment (SpaceLoveSs13) [reply code/game/objects/machinery/vending/marine_vending.dm:901]: My pr is to fix the broken lolipop pouches. Not balance change it. It was intended to be infinite. So I should not be touching it.

**Comment by BraveMole** [comment]
Just when i though you were cool, turns out you are a c*der

File: code/game/objects/machinery/vending/marine_vending.dm (hunk 1 above)
Comment (SplinterGP) [reply code/game/objects/machinery/vending/marine_vending.dm:901]: I believe those are not intended to be infinite only the ghhme ones should start full

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Pull request merged**